users/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+import json
+import bcrypt
+import jwt
+import re
+import requests
 
-# Create your views here.
+from django.views          import View
+from django.http           import JsonResponse
+
+from json.decoder          import JSONDecodeError
+
+from users.models          import User, UserInformation
+from my_settings           import SECRET_KEY, ALGORITHM
+
+
+class KakaoLogInView(View):
+    def get(self, request):
+        try:
+            access_token          = request.headers.get('Authorization', None)
+            headers               = {"Authorization" : f"Bearer {access_token}"}
+            kakao_login_data      = requests.get("https://kapi.kakao.com/v2/user/me", headers=headers).json()
+            kakao_account         = kakao_login_data["kakao_account"]
+            email                 = kakao_account.get("email")  
```

Comment:
가장 긴 변수를 기준으로 정렬하면 좋을 것 같아요
```suggestion
            access_token      = request.headers.get('Authorization', None)
            headers           = {"Authorization" : f"Bearer {access_token}"}
            kakao_login_data  = requests.get("https://kapi.kakao.com/v2/user/me", headers=headers).json()
            kakao_account     = kakao_login_data["kakao_account"]
            email             = kakao_account.get("email")  
```

Reply:
네 수정 반영하겠습니다.